pkg/services/apiserver/service.go

```diff
@@ -245,16 +254,23 @@ func (s *service) start(ctx context.Context) error {
 		if err := b.InstallSchema(s.scheme); err != nil {
 			return err
 		}
-		pvs := s.scheme.PrioritizedVersionsForGroup(gvs[0].Group)
+		_ = s.scheme.PrioritizedVersionsForGroup(gvs[0].Group)
 
-		for _, gv := range pvs {
-			if a, ok := b.(builder.APIGroupAuthorizer); ok {
-				auth := a.GetAuthorizer()
-				if auth != nil {
-					s.authorizer.Register(gv, auth)
-				}
+		if a, ok := b.(builder.APIGroupAuthorizer); ok {
+			auth := a.GetAuthorizer()
+			if auth != nil {
+				s.authorizer.Register(gvs[0], auth)
 			}
 		}
+
+		//for _, gv := range pvs {
+		//	if a, ok := b.(builder.APIGroupAuthorizer); ok {
+		//		auth := a.GetAuthorizer()
+		//		if auth != nil {
+		//			s.authorizer.Register(gv, auth)
+		//		}
+		//	}
+		//}
```

Comment:
This change has been done in [this commit](https://github.com/grafana/grafana/pull/105632/commits/f7f7c7853e05d964562adaea39689f24a5e397ad), but I have no idea why exactly it was needed.

Reply:
Without this change, when hitting ofrep endpoints I get 403:

```
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "ofrep.features.grafana.app \"v1\" is forbidden: User \"\" cannot create resource \"ofrep/evaluate\" in API group \"features.grafana.app\" in the namespace \"7414\"",
  "reason": "Forbidden",
  "details": {
    "name": "v1",
    "group": "features.grafana.app",
    "kind": "ofrep"
  },
  "code": 403
}
```